embulk run raise the error uninitialized constant Forwardable::INSTANCE

Issue Type: Bug Report
-----------------------

* Write the following environmental information.
  * OS version: macOS 10.15.7
  * Java version: 1.8.0_251
  * Embulk version: 0.10.22
  * Your Embulk configuration (YAML): guessed example file.
  * Plugin versions: builtin gzip, csv

## Write all what you did, e.g. your commands executed

`embulk run` raise the exception `uninitialized constant Forwardable::INSTANCE`
when JRuby 9.2.13.0 use

```
cat ~/.embulk-dev/embulk.properties
jruby=file:///path/to/.embulk/jars/jruby-complete-9.2.13.0.jar
```

```
embulk-dev -X embulk_home=/path/to/.embulk-dev gem install bundler -v 1.6.0
embulk-dev -X embulk_home=/path/to/.embulk-dev gem install msgpack -v 1.1.0
```

```
embulk-dev -X embulk_home=/path/to/.embulk-dev guess -o config.yml fuga/seed.yml
2020-12-01 14:07:00.833 +0900 [INFO] (main): embulk_home is set from command-line: /path/to/.embulk-dev
2020-12-01 14:07:01.452 +0900 [INFO] (main): Started Embulk v0.10.22
2020-12-01 14:07:01.571 +0900 [INFO] (0001:guess): Listing local files at directory '/private/tmp/fuga/csv' filtering filename by prefix 'sample_'
2020-12-01 14:07:01.571 +0900 [INFO] (0001:guess): "follow_symlinks" is set false. Note that symbolic links to directories are skipped.
2020-12-01 14:07:01.573 +0900 [INFO] (0001:guess): Loading files [/private/tmp/fuga/csv/sample_01.csv]
2020-12-01 14:07:01.584 +0900 [INFO] (0001:guess): Try to read 32,768 bytes from input source
2020-12-01 14:07:04.337 +0900 [INFO] (0001:guess): Gem's home and path are set by system configs "gem_home": "/path/to/.embulk-dev/lib/gems", "gem_path": ""
NameError: uninitialized constant Forwardable::INSTANCE
   const_missing at org/jruby/RubyModule.java:3760
  new_java_guess at uri:classloader:/META-INF/jruby.home/lib/ruby/stdlib/forwardable.rb:223
org.embulk.config.ConfigException: GuessPlugin 'gzip' is not found.
(NameError) uninitialized constant Forwardable::INSTANCE
	at org.embulk.plugin.PluginManager.buildPluginNotFoundException(PluginManager.java:71)
	at org.embulk.plugin.PluginManager.newPlugin(PluginManager.java:57)
	at org.embulk.spi.ExecSessionInternal.newPlugin(ExecSessionInternal.java:273)
	at org.embulk.spi.ExecInternal.newPlugin(ExecInternal.java:55)
	at org.embulk.exec.GuessExecutor$GuessParserPlugin.run(GuessExecutor.java:239)
	at org.embulk.spi.FileInputRunner.run(FileInputRunner.java:138)
	at org.embulk.exec.GuessExecutor$3.run(GuessExecutor.java:176)
	at org.embulk.spi.FileInputRunner$RunnerControl$1$1.run(FileInputRunner.java:111)
	at org.embulk.exec.GuessExecutor$GuessParserPlugin.transaction(GuessExecutor.java:224)
	at org.embulk.spi.FileInputRunner$RunnerControl$1.run(FileInputRunner.java:106)
	at org.embulk.spi.util.DecodersInternal$RecursiveControl.transaction(DecodersInternal.java:66)
	at org.embulk.spi.util.DecodersInternal.transaction(DecodersInternal.java:28)
	at org.embulk.spi.FileInputRunner$RunnerControl.run(FileInputRunner.java:104)
	at org.embulk.exec.BufferFileInputPlugin.transaction(BufferFileInputPlugin.java:21)
	at org.embulk.spi.FileInputRunner.transaction(FileInputRunner.java:61)
	at org.embulk.exec.GuessExecutor.guessParserConfig(GuessExecutor.java:171)
	at org.embulk.exec.GuessExecutor.guessParserConfig(GuessExecutor.java:151)
	at org.embulk.spi.FileInputRunner.guess(FileInputRunner.java:84)
	at org.embulk.exec.GuessExecutor.doGuess(GuessExecutor.java:127)
	at org.embulk.exec.GuessExecutor.access$000(GuessExecutor.java:41)
	at org.embulk.exec.GuessExecutor$2.run(GuessExecutor.java:100)
	at org.embulk.exec.GuessExecutor$2.run(GuessExecutor.java:97)
	at org.embulk.spi.ExecInternal.doWith(ExecInternal.java:26)
	at org.embulk.exec.GuessExecutor.guess(GuessExecutor.java:97)
	at org.embulk.EmbulkEmbed.guess(EmbulkEmbed.java:174)
	at org.embulk.EmbulkRunner.guessInternal(EmbulkRunner.java:200)
	at org.embulk.EmbulkRunner.guess(EmbulkRunner.java:57)
	at org.embulk.cli.EmbulkRun.runInternal(EmbulkRun.java:113)
	at org.embulk.cli.EmbulkRun.run(EmbulkRun.java:24)
	at org.embulk.cli.Main.main(Main.java:52)
	Suppressed: org.embulk.plugin.PluginSourceNotMatchException
		at org.embulk.plugin.InjectedPluginSource.newPlugin(InjectedPluginSource.java:51)
		at org.embulk.plugin.PluginManager.newPlugin(PluginManager.java:40)
		... 28 more
	Suppressed: org.embulk.plugin.PluginSourceNotMatchException
		at org.embulk.plugin.maven.MavenPluginSource.newPlugin(MavenPluginSource.java:69)
		at org.embulk.plugin.PluginManager.newPlugin(PluginManager.java:46)
		... 28 more
	Suppressed: org.embulk.plugin.PluginSourceNotMatchException: org.jruby.embed.InvokeFailedException: (NameError) uninitialized constant Forwardable::INSTANCE
		at org.embulk.jruby.JRubyPluginSource.newPlugin(JRubyPluginSource.java:88)
		at org.embulk.plugin.PluginManager.newPlugin(PluginManager.java:52)
		... 28 more
	Caused by: org.jruby.embed.InvokeFailedException: (NameError) uninitialized constant Forwardable::INSTANCE
		at org.jruby.embed.internal.EmbedRubyObjectAdapterImpl.call(EmbedRubyObjectAdapterImpl.java:320)
		at org.jruby.embed.internal.EmbedRubyObjectAdapterImpl.callMethod(EmbedRubyObjectAdapterImpl.java:159)
		at org.jruby.embed.ScriptingContainer.callMethod(ScriptingContainer.java:1447)
		at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
		at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
		at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
		at java.lang.reflect.Method.invoke(Method.java:498)
		at org.embulk.jruby.ScriptingContainerDelegateImpl.callMethod(ScriptingContainerDelegateImpl.java:634)
		at org.embulk.jruby.LazyScriptingContainerDelegate.callMethod(LazyScriptingContainerDelegate.java:233)
		at org.embulk.jruby.JRubyPluginSource.newPlugin(JRubyPluginSource.java:86)
		... 29 more
	Caused by: org.jruby.exceptions.NameError: (NameError) uninitialized constant Forwardable::INSTANCE
		at org.jruby.RubyModule.const_missing(org/jruby/RubyModule.java:3760)
		at RUBY.new_java_guess(uri:classloader:/META-INF/jruby.home/lib/ruby/stdlib/forwardable.rb:223)

Error: GuessPlugin 'gzip' is not found.
(NameError) uninitialized constant Forwardable::INSTANCE
```

--- a/embulk-core/src/main/ruby/embulk/plugin.rb
+++ b/embulk-core/src/main/ruby/embulk/plugin.rb
@@ -220,12 +220,12 @@ module Embulk
   end
 
   module Plugin
+    INSTANCE = PluginManager.new
     class <<self
-      INSTANCE = PluginManager.new
 
       extend Forwardable
 
-      def_delegators 'INSTANCE',
+      def_delegators :'Embulk::Plugin::INSTANCE',
         :register_input, :get_input, :register_java_input, :new_java_input,
         :register_output, :get_output, :register_java_output, :new_java_output,
         :register_filter, :get_filter, :register_java_filter, :new_java_filter,

